modules/http4s/src/smithy4s/http4s/internals/SmithyHttp4sClientEndpoint.scala

```diff
@@ -114,9 +114,12 @@ private[http4s] class SmithyHttp4sClientEndpointImpl[F[_], Op[_, _, _, _, _], I,
   def inputToRequest(input: I): Request[F] = {
     val metadata = inputMetadataEncoder.encode(input)
     val path = httpEndpoint.path(input)
+    val (extractedPath, queries) = splitPathSegmentsAndQueryParams(path)
```

Comment:
This doesn't seem to be checked by any tests

Reply:
what do you mean?
is this missing something https://github.com/disneystreaming/smithy4s/pull/692/files#diff-b475f48e311abe1d8cf9ca0c675210125637631a3ac2882afdc699794002dd4a

Reply:
I mean this line in `inputToRequest` - we're unit testing the function all right, but the usage isn't covered (testing the input->request conversion, or a full blown client test would help)